src/redux/modules/tags.js

```diff
@@ -0,0 +1,64 @@
+import { createAction, handleActions } from 'redux-actions';
+
+/* Types */
+
+const FETCH_TAGS         = 'Tags/FETCH_TAGS';
+const FETCH_TAGS_SUCCESS = 'Tags/FETCH_TAGS_SUCCESS';
+const FETCH_TAGS_FAILURE = 'Tags/FETCH_TAGS_FAILURE';
+
+export const types = {
+  FETCH_TAGS,
+  FETCH_TAGS_SUCCESS,
+  FETCH_TAGS_FAILURE,
+};
+
+/* Actions */
+
+const fetchTags        = createAction(FETCH_TAGS);
+const fetchTagsSuccess = createAction(FETCH_TAGS_SUCCESS);
+const fetchTagsFailure = createAction(FETCH_TAGS_FAILURE);
+
+export const actions = {
+  fetchTags,
+  fetchTagsSuccess,
+  fetchTagsFailure,
+};
+
+/* Reducer */
+
+const initialState = {
+  tags: {},
+  error: null,
+};
+
+const _fetchTagsSuccess = (state, action) => {
+  return {
+    ...state,
+    tags: action.payload,
+  };
+};
+
+const _fetchTagsFailure = (state, action) => {
+  return {
+    ...state,
+    error: action.payload,
+  };
+};
+
+export const reducer = handleActions({
+  [FETCH_TAGS_SUCCESS]: (state, action) => _fetchTagsSuccess(state, action),
```

Comment:
Need to drop all data and refetch on language change.
See example in [mdb module](https://github.com/Bnei-Baruch/kmedia-mdb/blob/master/src/redux/modules/mdb.js#L83)